### DISCUSSION TITLE
Suno AI Support

### DISCUSSION THREAD

**Discussion by vaibhavard** [Richard-Weiss/node-chatgpt-api#2] [General]
Support for SunoAI plugin already available in copilot Microsoft (allows you to compose your own music using bing chat)

**Comment by Richard-Weiss** [comment]
Plugins don't really work yet from my testing, so I haven't added it yet. But yeah, I could add it now so I won't forget.

**Reply by Richard-Weiss** [reply]
@vaibhavard Should be possible since I got the plugins to work. The Suno plugin requires some more work, but I think I found the necessary steps. See the linked issue for more information.

**Comment by Richard-Weiss** [comment]
@vaibhavard I think I'll actually postpone working on this for 2-3 weeks.
The implementation of Microsoft is quite odd an still a bit lackluster.
I don't want to have to redo it if they change it in a significant way, but you could still work on it through the way I describe in #3.
